transport/postis.js

```diff
@@ -84,12 +93,14 @@ function Postis(options) {
       var method = opts.method;
 
       if ((ready || opts.method === readyMethod) && (targetWindow && typeof targetWindow.postMessage === "function")) {
-        targetWindow.postMessage(JSON.stringify({
+        targetWindow.postMessage({
```

Comment:
isn't this a significant API change? how did this come about?

Reply:
or is this just handled by the matching listener?
curious, do we even need the stringify?

Reply:
yeah I've been wondering about stringify as well... it doesn't seem to be necessary really I think.
The API change doesn't matter for us because we deploy both at the same time (frontend loads external api from jitsi-meet deployment which will use the same version of this transport impl).

Reply:
so I found it by using "pause on caught exceptions" in chrome dev tools and it happens a lot while in local dev mode, it probably also happens any time any other lib sends a message, this client will parse it and abort on exception